spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -108,9 +110,10 @@ public TaskDefinitionResource create(String name, String definition) {
 	}
 
 	@Override
-	public void launch(String name, Map<String, String> properties) {
+	public void launch(String name, Map<String, String> properties, List<String> arguments) {
```

Comment:
sorry, removed wrong comments, called parameters as those go to 'java' arguments which are different than options. also in SPI those are called arguments to align same naming, but in shell params as requested.